ReleaseChecklist.md

```diff
@@ -39,6 +39,7 @@ At least a day before the release:
 
 ### Blog Post
  - [ ] Create a post on [solidity-website](https://github.com/ethereum/solidity-website/tree/main/src/posts) in the ``Releases`` category and explain some of the new features or concepts.
+       **Note**: This should include the update of [index.tsx](https://github.com/ethereum/solidity-website/blob/main/src/pages/index.tsx) with the new release version and description in the Solidity website main page.
```

Comment:
I always assumed it was covered by this:

> Update the release information section [in the source of soliditylang.org](https://github.com/ethereum/solidity-website/blob/main/src/pages/index.tsx).

Maybe we should just reword it to make it clearer?

Reply:
You're right, I'll close this one.